### ISSUE TITLE
Achieving a perfect accuracy on test set (quadrant split)

### ISSUE THREAD

**Issue by kojimano** [titu1994/keras-coordconv#9]
Using this implementation, it seems achiving the perfect auucracy in the test set for quadrant split is not as trivial as claimed in the paper. I also tested out the implemetation from a different repo, and they also had a same behaivor, so most likely this is not due to the problem in the implemantation. However, it would be great if people can share the expereince and tricks to achive an better accuracy in quadrant split.

**Comment by titu1994** [comment]
No tricks were used. It is however difficult and requires a good initialization. That's about it. I tried about 3 times, and the third time it trained to proper convergence. 

Other times, the Softmax scores were overlapping, so it was a grayish output rather than the sharp white.
